src/services/redisSessionStorage.ts

```diff
@@ -173,7 +207,8 @@ export default class RedisSessionStorage extends RedisStorageUtils
    */
   public async del(
```

Comment:
can we make del method to take `RequiredUserTokensV2` as input (an object instead of a tuple of tokens) ?

Reply:
This method must be usable with both `RequiredUserTokensV1` and `RequiredUserTokensV2` depends on the current status of the user session (the `myportal_token` is generated for a new session or when getSession is called).